### PR TITLE
Properly await graphql mutation for manual launch

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: utils/graphql.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const { gql, GraphQLClient } = require('graphql-request');
 const config = require('./config');
 const { default: { getChampsIdFromField, getAnnotationsIdFromField } } = require('../services/champsAndAnnotations');
+const { resolve } = require('../frontend/config/webpack.dev.hot');
```
Comment (sandrica89) [inline utils/graphql.js:4]: Wrong import

File: utils/graphql.js [hunk 2]
```diff
@@ -40,9 +41,12 @@ const request = async (query, variables) => {
   console.debug('GraphQL query sent:', query, variables);
 
   try {
-    const result = await graphQLClient.request(query, variables);
-    console.debug('GraphQL result: ', result);
-    return result;
+    const result = await graphQLClient
+    .request(query, variables).then((x) => {
+      console.log(x); resolve(x);
+    });
```
Comment (sandrica89) [inline utils/graphql.js:47]:
```suggestion
    const result = await graphQLClient.request(query, variables);
```

File: services/demarchesSimplifiees/autoAccept.js [hunk 3]
```diff
@@ -19,6 +19,8 @@ const sendAutoAcceptMessage = async (dossierId) => {
     .sendMessageWithAttachment(config.demarchesSimplifieesAutoAcceptMessage, uploadFileId, dossierId);
 
   console.log('message envoyé :', dossierEnvoyerMessage);
+
+  return Promise.resolve();
```
Comment (sandrica89) [inline services/demarchesSimplifiees/autoAccept.js:23]: I think this can be remove.

**Review by sandrica89 — CHANGES_REQUESTED** [review]
See PR comment

File: services/demarchesSimplifiees/autoAccept.js (hunk 3 above)
Comment (desoindx) [reply services/demarchesSimplifiees/autoAccept.js]: ooops

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**